src/crypto/pubkeyEncrypt.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { assert } from "chai";
+import {loadTestWallet, TestWallet} from "../testutils/wallets";
+import {PubkeyEncryptor} from "./pubkeyEncryptor";
+import {UserPublicKey, UserSecretKey} from "../userKeys";
+import {PubkeyDecryptor} from "./pubkeyDecryptor";
+import {X25519EncryptedData} from "./x25519EncryptedData";
+
+describe("test address", () => {
+    let alice: TestWallet, bob: TestWallet, carol: TestWallet;
+    const sensitiveData = Buffer.from("my secret text for x");
+    let encryptedData: X25519EncryptedData;
+
+    before(async () => {
+        alice = await loadTestWallet("alice");
+        bob = await loadTestWallet("bob");
+        carol = await loadTestWallet("carol");
+
+        encryptedData = PubkeyEncryptor.encrypt(sensitiveData, new UserPublicKey(bob.address.pubkey()), new UserSecretKey(alice.secretKey));
+    });
+
+    it("encrypts/decrypts",  () => {
```

Comment:
This step only include decryption, while encryption is written in `before()`. Was this intended? If so, perhaps rename `encryptedData` to `encryptedDataOfAliceForBob`?

Reply:
intended because I used the same "secret" for all tests - pulled it in the `before` hook to avoid code-duplication. Rename -> Done